lib/modules/platform/gitlab/index.spec.ts

```diff
@@ -2226,6 +2410,7 @@ describe('modules/platform/gitlab/index', () => {
         .reply(200, [])
         .post('/api/v4/projects/undefined/merge_requests/12345/approval_rules')
         .replyWithError('Unknown');
+      process.env.RENOVATE_X_GITLAB_AUTO_MERGEABLE_CHECK_ATTEMPS = '2';
```

Comment:
why do we need to set it so often? can't you set the default in `beforeEach` and only override on tests where required ?

Reply:
I refactored this part including the env variable `RENOVATE_X_GITLAB_MERGE_REQUEST_DELAY`. It was duplicated as well.